src/views/admin/general/sounds.tpl

```diff
@@ -53,22 +53,17 @@
 							<button type="button" class="form-control btn btn-sm btn-default" data-action="play">Play <i class="fa fa-play"></i></button>
 						</div>
 					</div>
+					
+					<div class="input-group col-xs-9">
```

Comment:
Well you do. It will look like this otherwise.

<img width="216" alt="screen shot 2016-05-07 at 12 41 18 pm" src="https://cloud.githubusercontent.com/assets/10014774/15094162/3752fdee-1451-11e6-9908-e41af9af258b.png">